api/models/orderDao.js

```diff
@@ -1,4 +1,69 @@
-const appDataSource = require('./datasource') 
+const appDataSource = require("./datasource")
+
+const deliveryInformationOfOder = async (userId, name, phoneNumber, address, arrivalDate, deliveryMethod) => {
+    const result = await appDataSource.query(
+        `INSERT INTO delivery_information(
+            users_id,
+            name,
+            phone_number,
+            address,
+            arrival_date,
+            delivery_method
+        )VALUES(?, ?, ?, ?, ?, ?)`,
+        [userId, name, phoneNumber, address, arrivalDate, deliveryMethod]
+    )
+    return result
+}
+
+const createOrder = async (deliveryInformationId, paymentMethodId, depositDeadline) => {
+    const result = await appDataSource.query(
+        `INSERT INTO orders(
+            delivery_information_id,
+            payment_method_id,
+            deposit_deadline,
+            order_status_id
+        )VALUES(?, ?, ?, 1)`,
+        [deliveryInformationId, paymentMethodId, depositDeadline]
+    )
+    return result
+}
+
+const orderProductsOfOder = async (optionProductsId, orderId, quantity) =>{
+    const result = await appDataSource.query(
+        `INSERT INTO order_products(
+            option_products_id,
+            order_id,
+            quantity,
+            option_products_status_id
+        )VALUES(?, ?, ?, 1)`,
+        [optionProductsId, orderId]
+    )
+    return result
+}
+
+
+const stockInOrder = async (optionProductsId, quantity) => {
```

Comment:
```suggestion
const getStockOfOptionProduct = async (optionProductsId, quantity) => {
```
하고자 하는 행위, 데이터를 한눈에 파악할 수 있게 작성해 주세요!